tests/decorators/test_decorators.py

```diff
@@ -267,6 +309,51 @@ def dummy():
                                  ]
                                  })
 
+    def test_service_bus_queue_trigger(self):
+        app = self.func_app
+
+        @app.on_service_bus_queue_change(arg_name="req",
+                                         connection="dummy_conn",
+                                         queue_name="dummy_queue")
+        def dummy():
+            pass
+
+        func = self._get_func(app)
+
+        trigger = func.get_bindings()[0]
+
+        self.assertEqual(trigger.get_dict_repr(), {
+            "direction": BindingDirection.IN,
+            "type": SERVICE_BUS_TRIGGER,
+            "name": "req",
+            "connection": "dummy_conn",
+            "queueName": "dummy_queue"
+        })
+
+    def test_service_bus_queue_output_binding(self):
+        app = self.func_app
+
+        @app.on_service_bus_queue_change(arg_name="req",
+                                         connection="dummy_conn",
+                                         queue_name="dummy_queue")
+        @app.write_service_bus_queue(arg_name='res',
+                                     connection='dummy_out_conn',
+                                     queue_name='dummy_out_queue')
+        def dummy():
+            pass
+
+        func = self._get_func(app)
+
+        trigger = func.get_bindings()[0]
```

Comment:
ditto, and change to output for any remaining tests

Reply:
I have renamed this for all tests as per recommendation. Thanks